src/migrations/003-permissions-vuex-to-composable.ts

```diff
@@ -0,0 +1,16 @@
+import type { Migration, PermissionRegistry } from '@/types';
+import { collectVuexState } from './migrationHelpers';
+
+const migration: Migration = async (restoredValue: PermissionRegistry) => {
+  if (!restoredValue || !Object.keys(restoredValue).length) {
+    const permissions = (await collectVuexState())?.permissions;
+    try {
+      if (permissions && Object.keys(permissions).length) {
+        return permissions;
+      }
+    } catch (error) { /* noop */ }
```

Comment:
Please introduce `handleUnknownError` in the catch block.

Reply:
It makes no sense to do anything as this is a migration. If user has the previous state corrupted we need to catch the error to avoid breaking the app but we do nothing with this fact.

Reply:
Any try catch block doing nothing at the catch is risky. It's hard to debug in the future.

Reply:
OK, I'll use `handleUnknownError`.